frontend/src/components/Banner.tsx

```diff
@@ -154,3 +106,46 @@ export const Banner = (props: BannerProps) => {
 
   return null;
 };
+
+type BannerCtaProps = {
+  target?: string;
+  content: string;
+  onClick?: () => void;
+  gaViewPing?: Parameters<typeof useGaViewPing>[0];
+  size?: "medium" | "large";
+} & ({ target: string } | { onClick: () => void }); // At least one of `target` and `onClick` is required:
+export const BannerCta = (props: BannerCtaProps) => {
+  const ctaRef = useGaViewPing(props.gaViewPing ?? null);
+
+  if (typeof props.target !== "string") {
+    return (
+      <div
+        className={
+          props.size === "large" ? styles["cta-large-button"] : styles.cta
+        }
+      >
+        <button onClick={props.onClick}>
+          <span ref={ctaRef}>{props.content}</span>
+        </button>
+      </div>
+    );
+  }
+
+  return (
+    <div
+      className={
+        props.size === "large" ? styles["cta-large-button"] : styles.cta
+      }
+    >
+      <OutboundLink
+        to={props.target}
+        eventLabel={props.content}
+        target="_blank"
+        rel="noopener noreferrer"
+        onClick={props.onClick}
+      >
+        <span ref={ctaRef}>{props.content}</span>
+      </OutboundLink>
+    </div>
+  );
+};
```

Comment:
non-blocking: I'd add some comments for this logic just to understand whats conditions are being checked to display which view.

Reply:
Good point, would help understanding what to change when in the future: a22e53b9